xclim/testing/tests/test_atmos.py

```diff
@@ -155,3 +155,22 @@ def test_high_precip_low_temp(pr_series, tasmin_series):
         pr, tas, pr_thresh="1 kg m-2 s-1", tas_thresh="1 C"
     )
     np.testing.assert_array_equal(out, [1])
+
+
+def test_wind_chill_index(atmosds):
+    out = atmos.wind_chill_index(ds=atmosds)
+
+    np.testing.assert_allclose(
```

Comment:
Expected values from the online calculator ? Just add a comment about where you got them.

Reply:
The online calculator rounds the output, so it's hard to use it directly here. I check by hand that xclim was returning values corresponding to the calculator, but the numbers themselves are self-computed, so it's more of a continuity test (to check if a change breaks it)...

Reply:
Ok found a way to alter the online calculator and remove the rounding. `test_indices` uses those expected values. The atmos test is left as-is.